ajax/libs/simplebar/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "simplebar",
+  "filename": "simplebar.min.js",
+  "description": "Scrollbars, simpler.",
```

Comment:
Let's use `Custom scrollbars with native scroll, done simple, lightweight, easy to use and cross-browser.` here, it's more comprehensive and also shown on GitHub and homepage.